graph.cpp

```diff
@@ -0,0 +1,98 @@
+#include <iostream>
+#include <list>
+#include <map>
+using namespace std;
```

Comment:
`using namespace std;` should be avoided, its considered bad practice, especially when working with multiple namespaces as you will be in this project (its likely you'll be working with a godot namespace, my API namespace, and the standard library namespace) Hence you should prefix all standard library objects and calls with the namespace `std::`.